src/Main.hs

```diff
@@ -42,44 +29,13 @@ fetchKeys opts = do
 
     else return ()
```

Comment:
This `if any... then do... else return ()` can be replaced by `when any... (do...)`.